From 9af84e1c08a2159c10baf13244e2c8a7d7e837e9 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 27 Jan 2022 23:02:20 -0800
Subject: [PATCH] Do not use NULL where boolean is expected

Fixes
src/GParted_Core.cc:73:57: error: static_cast from 'nullptr_t' to 'PedPartitionFlag' (aka '_PedPartitionFlag') is not allowed
|         for ( PedPartitionFlag flag = ped_partition_flag_next( static_cast<PedPartitionFlag>( NULL ) ) ;
|                                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
| 1 error generated.

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/GParted_Core.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/GParted_Core.cc b/src/GParted_Core.cc
index f7ac970..3290c66 100644
--- a/src/GParted_Core.cc
+++ b/src/GParted_Core.cc
@@ -70,7 +70,7 @@ GParted_Core::GParted_Core()
 	ped_exception_set_handler( ped_exception_handler ) ; 
 
 	//get valid flags ...
-	for ( PedPartitionFlag flag = ped_partition_flag_next( static_cast<PedPartitionFlag>( NULL ) ) ;
+	for ( PedPartitionFlag flag = ped_partition_flag_next( static_cast<PedPartitionFlag>( false ) ) ;
 	      flag ;
 	      flag = ped_partition_flag_next( flag ) )
 		flags .push_back( flag ) ;
-- 
2.35.0

